src/components/MoneyReportHeader.tsx

```diff
@@ -60,9 +60,10 @@ function MoneyReportHeader({session, policy, chatReport, nextStep, report: money
     const isAutoReimbursable = ReportUtils.canBeAutoReimbursed(moneyRequestReport, policy);
     const isPaidGroupPolicy = ReportUtils.isPaidGroupPolicy(moneyRequestReport);
     const isManager = ReportUtils.isMoneyRequestReport(moneyRequestReport) && session?.accountID === moneyRequestReport.managerID;
+    const isReimburser = session?.email === policy?.reimburserEmail;
```

Comment:
Might it be better to send the reimburser's accountID and compare it with session.accountID?

I think we standardised all to compare accountIDs. With comparing accountIDs, even if the reimburser changes their primary email, this check works fine. Reimbursers's email address can also remain hidden if we send only reimburser's accountID (though I don't think this is of much use.)

Reply:
Oh good point, the problem is that we're setting the reimburser email in the backend from OldDot. I agree we should compare accountIDs but that means we'd need to refactor also OldDot to save the accountID when the reimburser is selected. 

Said that, let's go for now with this change and we can refactor later to send the accountID when it becomes a problem or when the dashboard for policy settings is migrated to NewDot.

Reply:
@marcochavezf should we just update BE now to send the accountID? Or do we assume that all policy users should have access to this email info